dev/extra-ml-requirements.txt

```diff
@@ -4,35 +4,35 @@
 # Required by mlflow.azureml
```

Comment:
If it's too hard to get tests to pass after unpinning libraries in this file, perhaps we can merge the unpinning of MLflow's core / non-ML-extra requirements and then follow up with a PR to unpin extra ML requirements.